charts/linkerd-control-plane/templates/config.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
 data:
+  linkerd-crds-chart-version: linkerd-crds-1.0.0-edge
```

Comment:
Is this manually updated when the control plane starts depending on a linkerd-crds version different from this?